src/main/java/beans/AnswerQuestionRequestBean.java

```diff
@@ -31,21 +28,13 @@ public void setFormIndex(String formIndex) {
         this.formIndex = formIndex;
     }
     @JsonGetter(value = "answer")
-    public String getAnswer() {
+    public Object getAnswer() {
```

Comment:
this is either an array or a string right? may want to make a comment or a class specifying